README.md

```diff
@@ -584,6 +588,11 @@ db:
   # same as GRYPE_DB_UPDATE_URL env var
   update-url: "https://toolbox-data.anchore.io/grype/databases/listing.json"
 
+  # Max allowed age for vulnerability database,
+  # age being the time since it was built
+  # Default max age is 120h (or five days)
+  # Set it to 0 (zero) to disable check
+  max-allowed-db-age: "120h"
```

Comment:
since this is already under the `DB` section, to prevent a stutter renaming to `max-allowed-age` I think makes sense -- what's your take?

Reply:
I changed it to `max-allowed-built-age`.